### ISSUE TITLE
Stub docs in `/data` as Data Wrangling Guide

### ISSUE THREAD

**Issue by gsilvapt** [okfn/handbook#104]
Most of the content from the current `/data` directory is focused on tutorials and guides to Data Wrangling. Hence, the guides and tutorials should go to the Data Wrangling Guide.

**Comment by rufuspollock** [comment]
@gsilvapt but `/data/` **is** the data wrangling guide. At the moment we would just stub this - perhaps using the old base base from patterns (?).

Could you summarize in the description what would go on this page (note: if it was just stubbed it would be not much right now!)

**Comment by gsilvapt** [comment]
@rgrp The things inside this dir would be stubbed as `/data` to the Data Wrangling Guide, right?

**Comment by rufuspollock** [comment]
@gsilvapt i don't understand ;-) Data wrangling guide would like at `/data/`

```
/data/ - home page for data wrangling guide

/data/.../  # other pages in the guide

/data/patterns/ # data patterns
```

**Comment by gsilvapt** [comment]
@rgrp Is this issue fixed or obsolete with all your commits + archiving of some?

**Comment by rufuspollock** [comment]
FIXED.
